modules/visualizing-external-entities-known-limitations.adoc

```diff
@@ -0,0 +1,13 @@
+// Module included in the following assemblies:
+//
+// * operating/visualizing-external-entities.adoc
+:_mod-docs-content-type: CONCEPT
+[id="visualizing-external-entities-known-limitations_{context}"]
+= Known limitations
+
+[role="_abstract"]
+Following are some known limitations of the Visualizing external entities feature:
+
+* When you enable external IP collection for a cluster, Collector in those clusters reports more information to Sensor and to Central. It might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. Red{nbsp}Hat recommends that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
```

Comment:
We are supposed to [avoid _Red Hat recommends_](https://redhat-documentation.github.io/supplementary-style-guide/#recommend). Typically we use the _It is recommended_ structure (fwiw imho)
```suggestion
* When you enable external IP collection for a cluster, the Collector in those clusters report more information to Sensor and to Central. This might create scalability issues if the workload in the cluster communicates with a large number of distinct external peers. It is recommended that you do not enable this feature on clusters with communication patterns involving more than 10,000 distinct external entities.
```

Reply:
I don't think `the Collector` is correct. The documentation might sometimes say `the Collector image`, `the Collector container`, or `the Collector pod`, but never just `the Collector`. The other changes look good.

Reply:
Jouko is correct, we do not use "the" when referring to Collector, throughout the docs.